pysatNASA/instruments/methods/_cdf.py

```diff
@@ -1,6 +1,5 @@
 # -*- coding: utf-8 -*-
-"""Provides CDF class to parse cdaweb CDF files.
-"""
+"""Provides CDF class to parse cdaweb CDF files."""
```

Comment:
There are a few residual `string` as type in docstrings in the file. Most are `str`.